packages/framework/agent-scheduler/package.json

```diff
@@ -91,14 +91,8 @@
   },
   "typeValidation": {
     "version": "2.0.0-internal.2.2.0",
-    "baselineRange": "2.0.0-internal.2.0.0",
-    "broken": {
-      "InterfaceDeclaration_IAgentScheduler": {
-        "forwardCompat": false
-      },
-      "InterfaceDeclaration_IProvideAgentScheduler": {
-        "forwardCompat": false
-      }
-    }
```

Comment:
Looks fine, and so does the accompanying autogenerated tests file. @markfields [made `IAgentScheduler` implement `IFluidLoadable`](https://github.com/microsoft/FluidFramework/pull/12491) before we cut `internal-2.1.0`, but no changes between that one and `internal-2.2.0`.